simulator/mips/mips_instr.cpp

```diff
@@ -324,14 +350,20 @@ void FuncInstr::init( const FuncInstr::ISAEntry& entry)
                 << regTable[src2] << ", "
                 << std::dec << static_cast<int16>(v_imm);
             break;
+        case OUT_RI_BRANCH_0:
+            v_imm = instr.asI.imm;
+            src1 = static_cast<RegNum>(instr.asI.rs);
+            oss << " $" << regTable[src1] << ", "
+                << std::hex << "0x"
```

Comment:
Why hex here?

Reply:
For func bgez

Reply:
It should be decimal too.

Reply:
Instr bgzel too?

Reply:
Every instruction which starts with letter 'b'.

Reply:
Ok, I understand, all branches